docs/assets/js/script.js

```diff
@@ -45,19 +47,17 @@ devicon.controller('IconListCtrl', function($scope, $http, $compile) {
     $scope.svgBackground = DEFAULT_BACKGROUND;
 
     // Loop through devicon.json
-    angular.forEach(data, function(devicon, key) {
-
+    angular.forEach(data, function (devicon, key) {
       // New icon format
       var icon = {
         name: devicon.name,
         svg: devicon.versions.svg,
         font: devicon.versions.font,
-        main: ""
+        main: "",
```

Comment:
I prefer not having dangling `,` commas.
```suggestion
        main: ""
```